buildbot-host/scripts/build-deps.ps1

```diff
@@ -1,45 +1,38 @@
 param ([string] $workdir)
 
+. C:\Windows\Temp\ps_support.ps1
+
 Write-Host "Setting up openvpn build dependencies with vcpkg"
 
 if (-Not (Test-Path $workdir)) {
   New-Item -Type directory $workdir
+  CheckLastExitCode
 }
 
+Add-MpPreference -ExclusionPath $workdir
```

Comment:
Interesting, never had to use this. But if it is needed in some cases, so be it :smile:.

Reply:
It is not strictly required. It just means that the machine will spend a significant amount of its CPU capacity on running AV. If you have 2 CPUs, I usually see around 50% CPU going to AV during builds. If you have more cores, then it matters less. I will definitely leave it in the Jenkins code.